### DISCUSSION TITLE
Installazione Giua@school

### DISCUSSION THREAD

**Discussion by lucacaputo09** [iisgiua/giuaschool#683] [Richieste di aiuto]
Salve, dopo aver caricato la documentazione di giua su altervista come posso procedere all'installazione dell'account admin? Se è possibile spiegarlo passaggio per passaggio in modo da facilitarmi l'installazione

**Comment by trinko** [comment]
[Vedi qui](https://github.com/iisgiua/giuaschool/discussions/674#discussioncomment-8907284)
